pages/search/index.tsx

```diff
@@ -92,8 +96,28 @@ export default function Search({
     )
   }
 
+  const searchResultsHeadingRef = useRef(null)
+  useEffect(() => {
+    // don't focus on "Displaying n results..." if the page is not done loading
+    if (isLoading) return
+    // keep focus on sort by selector if the last update to the query was a sort
+    if (isFreshSortByQuery) {
```

Comment:
Small nit but can the parens be removed here?

Reply:
we love a single line if statement 💅🏻